bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -110,7 +110,7 @@ public AbstractBigtableAdmin(
     this.connection = connection;
     this.bigtableTableAdminClient = bigtableTableAdminClient;
     this.disabledTables = disabledTables;
-    this.clusterMetadataSetter = ClusterMetadataSetter.from(options);
+    this.bigtableClusterName = BigtableClusterName.from(options);
```

Comment:
nit: you could have the cluster name inside options, after all it is an inmutable configuration parameter. It would be created when options is created and you could avoid the "from" method indirection.